modules/virt-attaching-vm-secondary-network-cli.adoc

```diff
@@ -37,17 +37,20 @@ spec:
       networks:
         - name: default
           pod: {}
-        - name: bridge-net <1>
+        - name: bridge-net <2>
           multus:
-            networkName: <network-namespace/a-bridge-network> <2>
+            networkName: <network-namespace>/<a-bridge-network> <3>
 ...
 ----
-<1> The name of the bridge interface. This value must match the `name` value of the corresponding `spec.template.spec.networks` entry.
-<2> The name of the network attachment definition.
+<1> The name of the bridge interface.
+<2> The name of the bridge interface. This value must match the `name` value of the corresponding `spec.template.spec.networks` entry.
+<3> The name of the Network Attachment Definition, prefixed by the namespace where it exists. The only 2 possible options for this namespace are either the default namespace or the same namespace where the VM is to be created.
 +
 [NOTE]
 ====
-The example virtual machine is connected to both the `default` pod network and `bridge-net`, which is defined by a network attachment definition named `a-bridge-network`. If the network attachment definition is not in the same namespace as the VM, you must combine the namespace and the network name to create the network attachment definition name.
+The example virtual machine is connected to both the default pod network and the secondary network defined by a network attachment definition (NAD). The NAD must be created either in the same namespace as the VM or in the `default` namespace. However, if the current working namespace is different than the one where the NAD is created, the NAD name must be prefixed with the NAD namespace such `default/a-bridge-network`.
```

Comment:
Typo/suggestion: 

s/namespace such `default/a-bridge-network`./namespace. For example: `default/a-bridge-network`.

I'm confused about this note in general, though. The first two sentences are described in the preceding step and the callouts. The main point seems to be the third sentence, but the example yaml/callout seems to imply that you have to prefix it no matter what. Is there any harm in always using the prefix? To avoid confusion, I might remove the note.

Reply:
_but the example yaml/callout seems to imply that you have to prefix it no matter what_

Good point, I was actually wondering about the implication that the namespace prefix is mandatory, which derives from the yaml and callout. I decided to let it go, to avoid extra explaining and more confusion.
From my perspective, there's no harm in always using the namespace prefix. If you think it can indeed avoid the confusion, then yes - we can go with keeping the prefix in the yaml and callout as non-optional, and omit this last note, the limitation of using only `default` or the VM's namespace is already explained in the callout.